app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = True if not self.hidden else False
```

Comment:
You could simplify this condition by avoiding the if check as mentioned in the [checklist](https://github.com/annaminaieva/py-herbivores-and-carnivores/blob/develop/checklist.md). Use this approach in the future:
```suggestion
        self.hidden = not self.hidden
```